integration-cli/docker_cli_daemon_test.go

```diff
@@ -966,6 +966,36 @@ func (s *DockerDaemonSuite) TestDaemonLinksIpTablesRulesWhenLinkAndUnlink(c *che
 	s.d.Cmd("kill", "parent")
 }
 
+func (s *DockerDaemonSuite) TestDaemonNetworkPools(c *check.C) {
+	testRequires(c, DaemonIsLinux, SameHostDaemon)
+
+	// Remove docker0 bridge and the start daemon defining the predefined address pools
+	defaultNetworkBridge := "docker0"
+	deleteInterface(c, defaultNetworkBridge)
+	s.d.Start(c,
+		"--default-node-local-address-pools", "base=175.30.0.0/16,size=16",
+		"--default-node-local-address-pools", "base=175.33.0.0/16,size=24")
+
+	// Verify bridge network's subnet
+	out, err := s.d.Cmd("network", "inspect", "--format", "'{{.IPAM.Config}}'", "bridge")
```

Comment:
This can still be part of the API suite, and use the API to do the inspect instead of the docker cli.